src/v/kafka/server/connection_context.h

```diff
@@ -189,8 +189,9 @@ class connection_context final
      * The future<> returned by this method resolves when all ready *and*
      * in-order responses have been processed, which is not the same as all
      * ready responses. In particular, responses which are ready may not be
-     * processed if there are earlier (lower sequence number) responses which
-     * are not yet ready: they will be processed by a future invocation.
+     * processed if there are earlier (lower sequence number) responses
+     * which are not yet ready: they will be processed by a future
+     * invocation.
```

Comment:
Why the reflow?

Reply:
Ran clang fmt on everything. I can separate this.

Reply:
this also happens to me on occasion. the way this usually manifests for me is:

1. in editor i forget a `}` and clang format re-indents the entire file one level deep changing the text formatting in the process
2. i add back in the `}` and reformat, but the text formatting doesn't go back to exactly before.

Reply:
nit: I don't this change is necessary